src/css/style.css

```diff
@@ -28,6 +28,13 @@ body {
 #app {
   height: 95vh;
 }
+.center-block {
+  left: 50%;
+  top: 45%;
+  transform: translate(-50%, -50%);
+  max-width: 526px;
+  width: calc(100% - 40px - 40px);
```

Comment:
Not sure about this one

Reply:
The calc is for this.

![Captura de pantalla (1)](https://user-images.githubusercontent.com/71513679/173633429-c74749e5-01aa-4731-b7fc-bb1c81dcd33b.png)

Reply:
What about using padding? also 40px don't apply on all devices, please note that in design it applies to iPhone but what about the other devices?

Reply:
About the padding, the box-sizing property should solve this problem. And to make it work on all devices I will use percentages and media queries. Thanks for review

Reply:
Looks better for me! @victor-heliomar @angelik0828 what do you think?